### PR TITLE
BZ-1880759: Adding verification step for more than 3 etcd members

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://bugzilla.redhat.com/show_bug.cgi?id=1880759

Preview: https://deploy-preview-32579--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-etcd-member_replacing-unhealthy-etcd-member

### CODE REVIEW

**Comment by retroflexer** [comment]
LGTM

**Comment by bergerhoffer** [comment]
@geliu2016 Can you please review this update? Thanks!

Preview: https://deploy-preview-32579--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-etcd-member_replacing-unhealthy-etcd-member

File: modules/restore-replace-stopped-etcd-member.adoc [hunk 1]
```diff
@@ -350,4 +352,40 @@ If the output from the previous command only lists two pods, you can manually fo
 $ oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$( date --rfc-3339=ns )"'"}}' --type=merge <1>
 ----
 <1> The `forceRedeploymentReason` value must be unique, which is why a timestamp is appended.
+
+. Verify that there are exactly three etcd members.
+
+.. Connect to the running etcd container, passing in the name of a pod that was not on the affected node:
```
Comment (skrthomas) [inline modules/restore-replace-stopped-etcd-member.adoc:358]: s/passing in/specifying

File: modules/restore-replace-stopped-etcd-member.adoc [hunk 2]
```diff
@@ -350,4 +352,40 @@ If the output from the previous command only lists two pods, you can manually fo
...
++------------------+---------+------------------------------+---------------------------+---------------------------+
+|        ID        | STATUS  |             NAME             |        PEER ADDRS         |       CLIENT ADDRS        |
++------------------+---------+------------------------------+---------------------------+---------------------------+
+| 5eb0d6b8ca24730c | started |  ip-10-0-133-53.ec2.internal |  https://10.0.133.53:2380 |  https://10.0.133.53:2379 |
+| 757b6793e2408b6c | started |  ip-10-0-164-97.ec2.internal |  https://10.0.164.97:2380 |  https://10.0.164.97:2379 |
+| ca8c2990a0aa29d1 | started | ip-10-0-154-204.ec2.internal | https://10.0.154.204:2380 | https://10.0.154.204:2379 |
++------------------+---------+------------------------------+---------------------------+---------------------------+
+----
++
+If the output from the previous command lists more than three etcd members, you must carefully remove the unwanted member.
```
Comment (skrthomas) [inline modules/restore-replace-stopped-etcd-member.adoc:386]: s/unwanted member/unhealthy member...I see unhealthy used earlier, but maybe I am getting unhealthy and unwanted mixed up.

**Review by skrthomas — COMMENTED** [review]
several small suggestions (and questions?). looks good!! :)

File: modules/restore-replace-stopped-etcd-member.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/restore-replace-stopped-etcd-member.adoc:386]: Yeah by this point, the unhealthy member should be gone, so this is different. But I'm open to other synonyms of "unwanted"  - I couldn't think of anything better :)

File: modules/restore-replace-stopped-etcd-member.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/restore-replace-stopped-etcd-member.adoc:358]: The same language is used earlier in the procedure, but I'll consider whether to update them all in a future update. Thanks!

**Pull request merged**